src/KDGui/platform/linux/xcb/linux_xcb_platform_window.cpp

```diff
@@ -41,8 +41,9 @@ bool LinuxXcbPlatformWindow::create()
     const auto screen = m_platformIntegration->screen();
 
     m_xcbWindow = xcb_generate_id(connection);
-    uint32_t mask = XCB_CW_EVENT_MASK;
-    uint32_t values[1] = {
+    const uint32_t mask = XCB_CW_EVENT_MASK;
+    // NOLINTNEXTLINE(modernize-avoid-c-arrays)
```

Comment:
would it make sense to use std::array instead?

Reply:
Done